src/test/scala/zio/interop/reactivestreams/PublisherToStreamSpec.scala

```diff
@@ -130,6 +131,26 @@ object PublisherToStreamSpec extends DefaultRunnableSpec {
             fails(anything)
           )
         )
+      },
+      testM("ignores publisher calls after stream ending") {
+        withProbe(probe =>
+          assertM((for {
+            fiber  <- probe.toStream(bufferSize).runHead.fork
+            demand <- Task(probe.expectRequest())
+            _      <- Task(probe.sendNext(0))
+            _      <- Task(probe.expectCancelling())
+
+            _ <- TestClock.adjust(1.second)
```

Comment:
Do we really need time here? AFAIKS the test works without.

Reply:
oh, it's not needed, deleted it